### PR TITLE
feat: [sc-27017] RTE: Support Additional Embeds

### PR DESCRIPTION
**Pull request by ignatospadov**
Story details: https://app.shortcut.com/cheddar/story/27017

For reference, output html can be found in `src/mocks/html/contentful/` or here https://docs.google.com/spreadsheets/d/1rNMjgUjjcJuOXrBSYS2hWMEGoajll81WywrXsCIOS7Y/edit?usp=sharing

### CODE REVIEW

File: src/resolvers/contentful/test.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<iframe width="100%" height="300" scrolling="no" frameborder="no" allow="autoplay" src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/1438250014&color=%23ff5500&auto_play=false&hide_related=false&show_comments=true&show_user=true&show_reposts=false&show_teaser=true&visual=true"></iframe><div style="font-size: 10px; color: #cccccc;line-break: anywhere;word-break: normal;overflow: hidden;white-space: nowrap;text-overflow: ellipsis; font-family: Interstate,Lucida Grande,Lucida Sans Unicode,Lucida Sans,Garuda,Verdana,Tahoma,sans-serif;font-weight: 100;"><a href="https://soundcloud.com/karanaujla-music" title="Karan Aujla" target="_blank" style="color: #cccccc; text-decoration: none;">Karan Aujla</a> · <a href="https://soundcloud.com/karanaujla-music/take-it-easy" title="Take It Easy" target="_blank" style="color: #cccccc; text-decoration: none;">Take It Easy</a></div>
```
Comment (zhiln) [inline src/resolvers/contentful/test.html:1]: Besides that you're true fan of Karan Aujla, what's the reason of this snippet? 😆

File: src/resolvers/contentful/instagram.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import config from '../../config'
+
+export default (node) =>
```
Comment (zhiln) [inline src/resolvers/contentful/instagram.js:3]: Is there a particular reason to not to use iframe for Instagram?

**Review by zhiln — APPROVED** [review]
Looks great 👍  Left few comments.

File: src/resolvers/contentful/test.html (hunk 1 above)
Comment (ignatospadov) [reply src/resolvers/contentful/test.html:1]: oh shoot i didnt mean to commit this XD  nice catch

File: src/resolvers/contentful/instagram.js (hunk 2 above)
Comment (ignatospadov) [reply src/resolvers/contentful/instagram.js:3]: I was going based on the suggested implementaiton from instagram but now i see that we can totally use an iframe for this. nice catch!

**Review by zhiln — APPROVED** [review]
👍 👍 

Good job on readme file!

**Pull request merged**